src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -133,7 +132,6 @@ private async Task<Solution> ProcessResultAsync(CodeFixContext context, Diagnost
 
             var filteredLocations = fieldLocations.Filter(
                 (documentId, span) =>
-                    fieldDocument.Id == documentId &&
                     !span.IntersectsWith(declaratorLocation.SourceSpan) &&
```

Comment:
i don't get it.  this span check doesn't make sense if we're in different files.  how does that work?

Reply:
> how does that work?

Here the `fieldLocations` contains all locations found by renamer. This filter removes the original declaration from that list and does some additional filteing in `CanEditDocument`, that I didn't dive into much. Prior to this pr this was also filtering only references in the same document as the original field declaration, but since a partial class can be difined in multiple files, we can have references to that field from another file. Removing the ckeck allows to rename references to that field in other files, so they become property references later.

> this span check doesn't make sense if we're in different files

Uh, now I see what the original intent was. Let me change this, so this span check will happen only if we are in the same file